src/main/resources/assets/mmorpg/lang/zh_cn.json

```diff
@@ -918,488 +862,544 @@
-	"item.mmorpg.uniques.ring.ring_magic_shield0": "§e意志之塔",
+	"item.mmorpg.uniques.ring.ring_magic_shield0": "§e意志高塔",
 	"item.mmorpg.uniques.ring.ringcrit0": "§e精准戒指",
 	"item.mmorpg.uniques.ring.ringdodge0": "§e迅捷戒指",
-	"item.mmorpg.uniques.ring.ringenergy0": "§e无限耐久戒指",
-	"item.mmorpg.uniques.ring.ringwaterfire0": "§e冰霜火焰戒指",
+	"item.mmorpg.uniques.ring.ringenergy0": "§e无穷耐力戒指",
+	"item.mmorpg.uniques.ring.ringwaterfire0": "§e霜火戒指",
 	"item.mmorpg.uniques.ring.thunder_ele_ring0": "§e雷暴戒指",
 	"item.mmorpg.uniques.ring.water_ele_ring0": "§e暴雪戒指",
-	"item.mmorpg.uniques.shield.fire_ele_shield0": "§e火焰荆棘盾",
-	"item.mmorpg.uniques.shield.nature_ele_shield0": "§e毒刺盾",
-	"item.mmorpg.uniques.shield.shield_wisdom0": "§e智慧盾",
-	"item.mmorpg.uniques.shield.shieldresist0": "§e抗性盾",
-	"item.mmorpg.uniques.shield.thunder_ele_shield0": "§e雷电荆棘盾",
-	"item.mmorpg.uniques.shield.water_ele_shield0": "§e冰霜荆棘盾",
-	"item.mmorpg.uniques.staff.fire_ele_staff0": "§e火焰的安魂曲棒",
-	"item.mmorpg.uniques.staff.nature_ele_staff0": "§e毒液的安魂曲棒",
-	"item.mmorpg.uniques.staff.thunder_ele_staff0": "§e雷电的安魂曲棒",
+	"item.mmorpg.uniques.shield.fire_ele_shield0": "§e火焰荆棘盾牌",
+	"item.mmorpg.uniques.shield.nature_ele_shield0": "§e自然荆棘盾牌",
+	"item.mmorpg.uniques.shield.shield_wisdom0": "§e智慧盾牌",
+	"item.mmorpg.uniques.shield.shieldresist0": "§e抗性盾牌",
+	"item.mmorpg.uniques.shield.thunder_ele_shield0": "§e雷电荆棘盾牌",
+	"item.mmorpg.uniques.shield.water_ele_shield0": "§e冰霜荆棘盾牌",
+	"item.mmorpg.uniques.staff.fire_ele_staff0": "§e火焰的安魂曲权杖",
+	"item.mmorpg.uniques.staff.nature_ele_staff0": "§e自然的安魂曲权杖",
+	"item.mmorpg.uniques.staff.thunder_ele_staff0": "§e闪电的安魂曲权杖",
 	"item.mmorpg.uniques.staff.uniquestafffire0": "§e火山法杖",
-	"item.mmorpg.uniques.staff.uniquestafflifesteal0": "§e吸血法杖",
-	"item.mmorpg.uniques.staff.uniquestaffnature0": "§e灭世法杖",
+	"item.mmorpg.uniques.staff.uniquestafflifesteal0": "§e吸血鬼法杖",
+	"item.mmorpg.uniques.staff.uniquestaffnature0": "§e世界破坏者法杖",
 	"item.mmorpg.uniques.staff.uniquestaffthunder0": "§e雷暴法杖",
 	"item.mmorpg.uniques.staff.uniquestaffwater0": "§e永冻法杖",
-	"item.mmorpg.uniques.staff.water_ele_staff0": "§e冰霜的安魂曲棒",
-	"item.mmorpg.uniques.sword.fire_ele_saber0": "§e火焰疯狂剑",
-	"item.mmorpg.uniques.sword.nature_ele_saber0": "§e毒液疯狂剑",
+	"item.mmorpg.uniques.staff.water_ele_staff0": "§e冰霜的安魂曲权杖",
+	"item.mmorpg.uniques.sword.fire_ele_saber0": "§e火焰疯狂重剑",
+	"item.mmorpg.uniques.sword.nature_ele_saber0": "§e自然疯狂重剑",
 	"item.mmorpg.uniques.sword.swordnature0": "§e自然之灵剑",
-	"item.mmorpg.uniques.sword.swordphysical0": "§e厄运之刃",
+	"item.mmorpg.uniques.sword.swordphysical0": "§e末日之刃",
 	"item.mmorpg.uniques.sword.swordwater0": "§e冰元素剑",
-	"item.mmorpg.uniques.sword.thunder_ele_saber0": "§e雷电疯狂剑",
-	"item.mmorpg.uniques.sword.water_ele_saber0": "§e冰霜疯狂剑",
-
-"_comment": " [CATEGORY]: UNIQUE ITEMS - NAMES",
-
-
-"_comment": " [CATEGORY]: GEAR SLOTS - NAMES",
-
+	"item.mmorpg.uniques.sword.thunder_ele_saber0": "§e雷电疯狂重剑",
+	"item.mmorpg.uniques.sword.water_ele_saber0": "§e冰霜疯狂重剑",
```

Comment:
Usually the Unique gear means a story, that story is the reason make this gear so unique and powerful. so the translator need to zarefully select some formal words to show its "Unique". The old translation, in my oppinion, doesn't reach the point.